.env.example

```diff
@@ -34,7 +34,7 @@ NEXT_PUBLIC_ACTIVE_DELEGATE_MINT=false
 NEXT_PRIVATE_DELEGATE_SEED_PHRASE="add you seed phrase here only for delegate purpose"
 ## Public address
 NEXT_PUBLIC_DELEGATE_ADDRESS="0xaddyouraddresshereonlyfordelegatepurpose"
-## Add delegate when mint for address
+## Add a delegator address when mint for address
 NEXT_PUBLIC_ADD_DELEGATE_MINT_ADDRESS="false"
```

Comment:
# automatically activate delegation to BuilderPlace when minting a TLid
NEXT_PUBLIC_ACTIVATE_DELEGATE_ON_MINT